tests/sanitizer/ubsan_overflow_same_place.c

```diff
@@ -0,0 +1,13 @@
+int main(int argc, char **argv) {
```

Comment:
Can you put these files under "tests/core"?  And can you name them exactly that same as the test itself.  e.g. `test_ubsan_minimal_errors_same_place`

Reply:
> Can you put these files under "tests/core"? And can you name them exactly that same as the test itself. e.g. `test_ubsan_minimal_errors_same_place`